app/src/componentLibrary/attributeList/attributeList.scss

```diff
@@ -23,7 +23,7 @@
 .attributes {
   display: inline-flex;
   flex-wrap: wrap;
-  gap: 16px;
+  gap: 8px;
```

Comment:
Why do we change the gap between attributes?

Reply:
according to design